tests/testthat/test-tait-bryan.R

```diff
@@ -1,11 +1,19 @@
+test_that("unspecified unit argument throws error", {
+  expect_error(tait_bryan(yaw = 0, pitch = 0, roll - 0))
```

Comment:
I think this should be "roll=0" - turns out you still get the error anyways, but that's kind of a quirk of R's evaluation techniques.